odoo/tools/translate.py

```diff
@@ -1638,10 +1640,14 @@ def _get_translation_upgrade_queries(cr, field):
                 lang: field.translate(terms_mapping.get, src_value)
                 for lang, terms_mapping in translations.items()
             }
+            missing_languages = languages - translations.keys()
             if "en_US" not in new_values:
                 new_values["en_US"] = field.translate(lambda v: None, src_value)
             if extra and extra[0] not in new_values:
                 new_values[extra[0]] = new_values["en_US"]
```

Comment:
And I think if you fill all missing languages, you don't need commit 2575dff9362ba6dd2a97db8911ac8b867b126e0

Reply:
I was also thinking about that but for a website, we are adding translation for just the default language (all customers are not using all languages on the website and we can identify one specific language for the website) so I think that commit is also okay and now we have added this https://github.com/odoo/odoo/pull/135760/files#diff-9f9f8f7fdb4edf151cf1ba03efc0e5dd7ea6857d431d5a6f80fe769977ac7161R1647